lib/authentication/auth_okta.js

```diff
@@ -165,10 +165,16 @@ function AuthOkta(connectionConfig, httpClient) {
   * @returns {null}
   */
   function validateURLs(authenticator, ssoUrl, tokenUrl) {
-    authenticator = authenticator.toLowerCase();
-    if (!(authenticator.startsWith(ssoUrl.substring(0, authenticator.length)) &&
-      authenticator.startsWith(tokenUrl.substring(0, authenticator.length)))) {
-      throw new Error('The prefix of the SSO/token URL and the specified authenticator do not match.');
+    try {
+      const aUrl = new URL(authenticator.toLowerCase());
```

Comment:
It is worth to check because probably for creating  new URL case sensitive isn't important